src/db/mod.rs

```diff
@@ -18,11 +18,10 @@ pub(crate) mod file;
 mod migrate;
 
 /// Connects to database
-pub fn connect_db() -> Result<Connection, Error> {
+pub fn connect_db() -> Result<Connection, failure::Error> {
     // FIXME: unwrap might not be the best here
-    let db_url = env::var("CRATESFYI_DATABASE_URL")
-        .expect("CRATESFYI_DATABASE_URL environment variable is not exists");
```

Comment:
It'd be good to keep this message via `.context` or something, [`VarError::NotPresent`](https://doc.rust-lang.org/nightly/std/env/enum.VarError.html#variant.NotPresent) doesn't carry the variable name or anything useful.